Modules/_fuzz/_fuzzmodule.c

```diff
@@ -0,0 +1,52 @@
+#include <Python.h>
```

Comment:
Sure, but this would be like 2 lines in Cython. It's just the typical boilerplate of extension modules, nothing to write home about, right?

We don't need it. The thing we _need_ is to make sure that the fuzz tests are maintained and don't break -- for most ways it could break, it would be enough just to build them, but not run them. I think it is a good idea to also run them. Although there's a cost: the macro silliness to make them runnable individually and collectively probably makes it easy to screw up, so I'm not happy about it. (Every fuzz test name is repeated like four times.)  The strategy you mentioned above would make it less error prone.